sdk/browser/test/unit/PlatformEncryptionTools.test.ts

```diff
@@ -47,4 +49,14 @@ describe('PlatformEncryptionTools', () => {
 
     expect(response).to.eql(badEncryptedDataObject)
   })
+
+  it('#signWithPrivateKey', async () => {
```

Comment:
maybe also add an additional test that will check that two calls to `signWithPrivateKey` with the same arguments will result in the same value